avionics/common/include/states/ascent_to_apogee.h

```diff
@@ -9,20 +9,29 @@ namespace State {
 
 class AscentToApogee : public IState {
   public:
+    /**
+     * @brief Construct a new state object
+     * @param post_apogee_id
+     * @param mach_lock_id
+     * @param APOGEE_CHECKS number of apogee checks required before drogue_ignitor_.fire()
+     * @param MACH_LOCK_CHECKS number of mach lock checks required before next state
+     * @param MACH_LOCK_VELOCITY minimum velocity required before next state
```

Comment:
I'd say before entering MACH_LOCK in these two lines because this would be a specific case